src/status_im/ui/screens/chat/components/reply.cljs

```diff
@@ -47,21 +66,63 @@
              literal))
          parsed-text)))
 
-(defn reply-message [{:keys [from]}]
+(defn reply-message-old [{:keys [from]}]
   (let [contact-name       @(re-frame/subscribe [:contacts/contact-name-by-identity from])
         current-public-key @(re-frame/subscribe [:multiaccount/public-key])]
     [rn/view {:style {:flex-direction :row}}
-     [rn/view {:style (styles/reply-content)}
+     [rn/view {:style (styles/reply-content-old)}
       [quo/text {:weight          :medium
                  :number-of-lines 1
                  :style           {:line-height 18}}
-       (format-reply-author from contact-name current-public-key)]]
+       (format-reply-author-old from contact-name current-public-key)]]
      [rn/view
       [pressable/pressable {:on-press            #(re-frame/dispatch [:chat.ui/cancel-message-reply])
                             :accessibility-label :cancel-message-reply}
        [icons/icon :main-icons/close-circle {:container-style (styles/close-button)
                                              :color (:icon-02 @quo.colors/theme)}]]]]))
 
+(defn reply-message [{:keys [from identicon content-type contentType parsed-text content]} in-chat-input?]
+  (let [contact-name       @(re-frame/subscribe [:contacts/contact-name-by-identity from])
+        current-public-key @(re-frame/subscribe [:multiaccount/public-key])
+        content-type       (or content-type contentType)]
+    [rn/view {:style {:flex-direction :row :height 24}}
+     [rn/view {:style (styles/reply-content)}
+      [icons/icon :main-icons/connector {:color (quo2.colors/theme-colors quo2.colors/neutral-40 quo2.colors/neutral-60)
```

Comment:
Do you think it might be better to destructure the namespaces in `:require` ?
instead of having  `quo2.colors/theme-colors`
We could have a require that says
```
[quo2.colors :as colors]. or [quo2.colors :refer [theme-colors]]
```
And we can only use `(theme-colors)` or `(colors/theme-colors)`

Reply:
For colors namespace we need to differenciate from `quo` and `quo2` colors. Having just `colors` would make code a bit confusing, at least at this stage were we are maintaining both design systems. I suppose that when we fully migrate to quo2 we could do that.

But just added the `theme-colors` destructuring :) Thanks for the suggestion

Reply:
I see now, Cool